sklearn/ensemble/_gb_losses.py

```diff
@@ -527,6 +527,8 @@ def _raw_prediction_to_proba(self, raw_predictions):
         -------
         probas : 2d array, shape (n_samples, K)
             The predicted probabilities.
+
+        .. versionadded:: 0.20
```

Comment:
is this intentionally dating the `_raw_prediction_to_proba` method? As a private method, I don't think it benefits from such a label, which is focused on public API versioning